packages/components/src/components/decorations/highlight-decoration.svg

```diff
@@ -0,0 +1,30 @@
+<svg width="895" height="674" viewBox="0 0 895 674" fill="#86B6C1" class="firefox-highlight-fix">
```

Comment:
was supposed to be just transparent for this svg?

Reply:
I see the change in storybook, nice! 
<img width="1069" alt="image" src="https://github.com/user-attachments/assets/f92c5e26-64c3-4f05-b6cb-3cf96910da8a">